RevertRebrand.theme.css

```diff
@@ -6,7 +6,7 @@
   * @invite neMncS2
   * @source https://Goose-Nest.github.io/GT-RevertRebrand/src/RevertRebrand.theme.css
   * @website https://github.com/Goose-Nest/GT-RevertRebrand
-  * @version 5.2
+  * @version 6.0
```

Comment:
Please don't bump the version in this PR, thanks

Reply:
Changed version number back to old one